### PR TITLE
network: add support for vlan over bond to stage 2 kickstart (pre)

### PR DESCRIPTION
**Pull request by rvykydal**
Required to move to kickstart processing in stage 2.

### CODE REVIEW

**Review by M4rtinK — APPROVED** [review]
Looks good to me. :)

File: pyanaconda/modules/network/nm_client.py [hunk 1]
```diff
@@ -352,6 +367,18 @@ def _update_wired_connection_with_s390_settings(connection, s390cfg):
         s_wired.props.s390_options = opts_dict
 
 
+def _create_new_connection(network_data, device_name):
+    con_uuid = NM.utils_uuid_generate()
+    con = NM.SimpleConnection.new()
+    s_con = NM.SettingConnection.new()
+    s_con.props.uuid = con_uuid
+    s_con.props.id = device_name
+    s_con.props.interface_name = device_name
```
Comment (VladimirSlavik) [inline pyanaconda/modules/network/nm_client.py:376]: I don't know much about netowrking, but won't naming a connection same as the device be confusing?

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Review by poncovka — APPROVED** [review]
Looks good to me.

**Comment by VladimirSlavik** [comment]
Jenkins, test this please.

File: pyanaconda/modules/network/nm_client.py (hunk 1 above)
Comment (rvykydal) [reply pyanaconda/modules/network/nm_client.py:376]: This is a convention we've been sticking to for quite some time in the installer.

**Pull request merged**